src/logic/authorizations/types.ts

```diff
@@ -0,0 +1,41 @@
+import type { LinkerAuthorization } from '@dcl/schemas'
+
+export type Authorizations = LinkerAuthorization[]
```

Comment:
Why do we need to export this type? We are not using it anywhere else.

Reply:
You're right, removed it!